src/winetricks

```diff
@@ -6269,7 +6271,8 @@ load_dotnet30()
     WINEDLLOVERRIDES="ngen.exe,mscorsvw.exe=b;$WINEDLLOVERRIDES"
 
     w_try_cd "$W_CACHE/$W_PACKAGE"
-    w_try "$WINE" "$file1" ${W_OPT_UNATTENDED:+ /q /c:"install.exe /q"}
+    w_warn "Installing .NET 3.0 runtime silently as otherwise it gets hidden behind taskbar."
```

Comment:
Maybe "Installing .NET 3.0 runtime silently, as otherwise it gets hidden behind taskbar.  Installation usually takes about 3 minutes."